<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerTractographyDisplayWidget</class>
 <widget class="qSlicerWidget" name="qSlicerTractographyDisplayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>779</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="VisibilityCheckBox">
     <property name="styleSheet">
      <string notr="true">QCheckBox::indicator {
     width: 21px;
     height: 21px;
 }

 QCheckBox::indicator:checked {
     image: url(:/Icons/Medium/SlicerVisible.png);
 }
QCheckBox::indicator:unchecked {
     image: url(:/Icons/Medium/SlicerInvisible.png);
 }</string>
     </property>
     <property name="text">
      <string>Visibility</string>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="OpacityLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Opacity:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QGroupBox" name="GroupBox_ColorBy">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Color By:</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="GroupBox_ColorByScalars">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Fiber Bundle Coloring</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0" colspan="2">
          <widget class="qMRMLColorTableComboBox" name="ColorByScalarsColorTableComboBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="renameEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <layout class="QHBoxLayout" name="Layout_TensorSelect">
           <item>
            <widget class="QLabel" name="ActiveTensorLabel">
             <property name="text">
               <string>Active Tensor:</string>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="ctkVTKDataSetArrayComboBox" name="ActiveTensorComboBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="attributeTypes">
              <set>ctkVTKDataSetModel::NoAttribute|ctkVTKDataSetModel::TensorsAttribute</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0" colspan="2">
          <layout class="QHBoxLayout" name="Layout_ScalarInvariantPropertySelect">
           <item>
            <widget class="QRadioButton" name="ColorByScalarInvariantRadioButton">
             <property name="text">
              <string>Of Tensor Property</string>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string>buttonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="ColorByScalarInvariantComboBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0" colspan="2">
          <layout class="QHBoxLayout" name="Layout_ScalarPropertySelect">
           <item>
            <widget class="QRadioButton" name="ColorByScalarRadioButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Of Scalar Value</string>
             </property>
             <attribute name="buttonGroup">
              <string>buttonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="ctkVTKDataSetArrayComboBox" name="ColorByScalarComboBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="attributeTypes">
              <set>ctkVTKDataSetModel::NoAttribute|ctkVTKDataSetModel::ScalarsAttribute</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QRadioButton" name="ColorByMeanFiberOrientationRadioButton">
           <property name="text">
            <string>Color Fibers By Mean Orientation</string>
           </property>
           <attribute name="buttonGroup">
            <string>buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QRadioButton" name="ColorByPointFiberOrientationRadioButton">
           <property name="text">
            <string>Color Fibers By Segment Orientation</string>
           </property>
           <attribute name="buttonGroup">
            <string>buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="6" column="0" colspan="2">
          <widget class="QRadioButton" name="ColorByCellScalarsRadioButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Of Fiber Cluster Group ID</string>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string>buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="7" column="0" colspan="2">
          <layout class="QHBoxLayout" name="Layout_SolidColorPropertySelect">
           <item>
            <widget class="QRadioButton" name="ColorBySolidColorRadioButton">
             <property name="text">
              <string>Solid Color</string>
             </property>
             <attribute name="buttonGroup">
              <string>buttonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="ctkColorPickerButton" name="ColorBySolidColorPicker"/>
           </item>
          </layout>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="ScalarRangeLabel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Set range:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QCheckBox" name="AutoWL">
           <property name="text">
            <string>Auto W/L</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0" colspan="2">
          <widget class="qMRMLRangeWidget" name="FiberBundleColorRangeWidget">
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="maximumValue">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="ctkCollapsibleGroupBox" name="MaterialPropertyGroupBox">
        <property name="title">
         <string>&amp;Material Properties</string>
        </property>
        <property name="collapsed">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="ctkMaterialPropertyWidget" name="MaterialPropertyWidget"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="ctkSliderWidget" name="OpacitySlider">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLColorTableComboBox</class>
   <extends>qMRMLNodeComboBox</extends>
   <header>qMRMLColorTableComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLRangeWidget</class>
   <extends>ctkRangeWidget</extends>
   <header>qMRMLRangeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkVTKDataSetArrayComboBox</class>
   <extends>QComboBox</extends>
   <header>ctkVTKDataSetArrayComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkColorPickerButton</class>
   <extends>QPushButton</extends>
   <header>ctkColorPickerButton.h</header>
  </customwidget>
  <customwidget>
   <class>ctkMaterialPropertyWidget</class>
   <extends>QWidget</extends>
   <header>ctkMaterialPropertyWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ctkRangeWidget</class>
   <extends>QWidget</extends>
   <header>ctkRangeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>VisibilityCheckBox</tabstop>
  <tabstop>ColorByScalarInvariantRadioButton</tabstop>
  <tabstop>ColorByScalarInvariantComboBox</tabstop>
  <tabstop>ColorByScalarRadioButton</tabstop>
  <tabstop>ColorByScalarComboBox</tabstop>
  <tabstop>ColorByCellScalarsRadioButton</tabstop>
  <tabstop>ColorByMeanFiberOrientationRadioButton</tabstop>
  <tabstop>ColorByPointFiberOrientationRadioButton</tabstop>
  <tabstop>ColorBySolidColorRadioButton</tabstop>
  <tabstop>ColorBySolidColorPicker</tabstop>
  <tabstop>MaterialPropertyGroupBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerTractographyDisplayWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ColorByScalarsColorTableComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>VisibilityCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>OpacitySlider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ColorByMeanFiberOrientationRadioButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>ColorByScalarsColorTableComboBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ColorByPointFiberOrientationRadioButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>ColorByScalarsColorTableComboBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ColorByScalarRadioButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>ColorByScalarsColorTableComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
